docs/functions.rst

```diff
@@ -1573,6 +1573,41 @@ Example:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
 
+DeleteDataUsingKopiaServer
+----------
+
+This function deletes the snapshot data backed up by the ``BackupDataUsingKopiaServer`` function.
+It creates a new Pod that runs ``delete snapshot`` job created using Kopia Repository Server
```

Comment:
```
It creates a new Pod that runs ``delete snapshot`` job created using Kopia Repository Server
```
I think this sentence is incomplete? I am not able to understand what this means?

Reply:
Will look into it !